### DISCUSSION TITLE
Atmos playing

### DISCUSSION THREAD

**Discussion by Krankig** [tidal-music/discussions#63] [Q&A]
Hi,
I am wondering how to play ATMOS music from TIDAL on Marantz receiver? I didn't find any answer that is really simple and clear.
I even bought a Fire Stick from Amazon but the sound is not as good as HEOS in stereo so? Who can help me please?
Thank you,

**Comment by stefanmoro** [comment]
Hey @Krankig ,

This GitHub discussions forum aims at handling technical discussions around the [TIDAL Developer Platform](https://developer.tidal.com/) (TIDAL API/SDK, open source projects etc).

As this question relates to an existing product, I will kindly ask you to reach out to our main [TIDAL support](https://support.tidal.com/).
